### PR TITLE
[readme] fix README file

### PR DESCRIPTION
**Pull request by Akasurde**
Fixes: #235
Fixes: #236

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+
+## Quick demo
 
 [![asciicast](https://asciinema.org/a/gl7uVblC23dxGGTkVOEigDHCl.svg)](https://asciinema.org/a/gl7uVblC23dxGGTkVOEigDHCl)
 
-## Quick start
+## Installation
 
 ```
 mkdir project_directory
```
Comment (ganeshrn) [inline README.md:12]: Add `cd project_directory`

File: README.md [hunk 2]
```diff
@@ -16,102 +18,116 @@ pip install ./ansible-navigator
...
-By default, ansible-navigator uses execution environments, to use ansible-navigator without an execution enviroment,
-ansible is required
+By default, `ansible-navigator` uses execution environments. Execution environments requires container runtimes like [podman](https://github.com/containers/podman). To install container runtimes, please refer respective installation guides.
+
+
+To use `ansible-navigator` without an execution enviroment, Ansible is required.
 
 ```
 pip install ansible
 ansible-navigator --execution-environment false
```
Comment (ganeshrn) [inline README.md:28]: This command is not required as it doesn't set EE to false for a subsequent run. To disable EE you point to the configuration doc and set `enabled: Fasle` and can point to https://github.com/ansible/ansible-navigator/blob/main/docs/settings.rst

File: README.md [hunk 3]
```diff
@@ -16,102 +18,116 @@ pip install ./ansible-navigator
...
 ```
 
 
-RHEL8/Centos8 prerequisites:
+RHEL 8 / CentOS 8 prerequisites:
```
Comment (ganeshrn) [inline README.md:32]: This section can be added at the top

File: README.md [hunk 4]
```diff
@@ -16,102 +18,116 @@ pip install ./ansible-navigator
...
 ansible-navigator
 ```
 
+From the welcome page you can run playbooks, browse collections, explore inventories, review docs, and do a lot more things.
+
+
 ### Other things to try direct from the command line
 
-#### Using interactive mode, which is the default
+#### Using interactive mode (the default mode)
```
Comment (ganeshrn) [inline README.md:53]: Maybe `(the default UI mode)`

File: README.md [hunk 5]
```diff
@@ -16,102 +18,116 @@ pip install ./ansible-navigator
...
-ansible-navigator can be configured:
+`ansible-navigator` can be configured:
 
-1) using default values
-2) with a settings file
-3) with environment variables
-4) at the command line
-5) while issuing `:` comamnds within the text-based user interface (TUI)
+* Using default values
+* With a configruation file
```
Comment (ganeshrn) [inline README.md:120]: Change this to `With a settings file`

File: README.md [hunk 6]
```diff
@@ -16,102 +18,116 @@ pip install ./ansible-navigator
...
+* With environment variables
+* By passing parameters at the command line
+* While issuing `:` comamnds within the text-based user interface (TUI)
```
Comment (ganeshrn) [inline README.md:123]: Fix typo in `commands`

File: README.md (hunk 2 above)
Comment (Akasurde) [reply README.md:28]: ok.

**Pull request closed without merging**